### DISCUSSION TITLE
Right klik on windows key not working

### DISCUSSION THREAD

**Discussion by robbyappel** [valinet/ExplorerPatcher#1348] [Q&A]
I have ep version 5.45 on my windows 11 (22H2 22621.1631  KB5025305), and the Right klik on the windows key is not working anymore. no menu is shown.
I get it back working again when i deinstall the EP.

Is this about a setting in EP or is it a bug ?

**Comment by topia** [comment]
What is "Right Click" on the "WIndows key"? If you mentioned right-clicking on the Start (Windows) "Button", it's not a supported feature for symbol-less builds.
https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Comment by just4t** [comment]
Same issue here  after upgrading to 10.0.22621.1635 due to symbols for that version were not downloaded automatically afterwards. Then, _Advanced | Enable Console_ helped me to force its symbols to be downloaded & installed. Thanks @topia for the given guidelines and [link](https://github.com/valinet/ExplorerPatcher/wiki/Symbols).

**Comment by robbyappel** [comment]
Yes, i have the preview version (V 22H2 KB5025305) of Windows11 installed this morning, and the right Clik on the start button is working again on EP version 22621.1555.55.1
